style.css

```diff
@@ -0,0 +1,610 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0px;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 1320px;
+  position: relative;
+}
+
+.container_top {
+  position: relative;
+  height: 670px;
+  background-color: #f6f9fb;
+  z-index: 5;
+}
+
+.bubble21 {
+  position: absolute;
+  height: 780px;
+  right: 0px;
+  top: -30px;
+}
+
+.banner {
+  position: absolute;
+  width: 900px;
+  right: -40px;
+  top: 120px;
+}
+
+.baner_text {
+  position: absolute;
+  padding: 20px;
+  width: 490px;
+  font-size: 15px;
```

Comment:
Polecam się zapoznać z jednostką `rem` i `em` jeśli chodzi o rozmiar tekstu. Dotyczy to całego pliku.